indexing-service/src/main/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTask.java

```diff
@@ -66,6 +72,20 @@
  * The client representation of this task is {@link ClientKillUnusedSegmentsTaskQuery}.
  * JSON serialization fields of this class must correspond to those of {@link
  * ClientKillUnusedSegmentsTaskQuery}, except for {@link #id} and {@link #context} fields.
+ * <br/>
+ * <br/>
+ * The Kill task fetches the set of used segments for the interval and computes the set of their load specs. <br/>
+ * Until `limit` segments have been processed in total or all segments for the interval have been nuked: <br/>
```

Comment:
Line break not needed here as lists start in a new line anyway.
```suggestion
 * Until `limit` segments have been processed in total or all segments for the interval have been nuked:
```

Reply:
Done